eap-cluster-app/build.sh

```diff
@@ -19,10 +19,10 @@ case "$1" in
       #Unzipping the JDG EAP Modules to jdg-modules directory
       echo "Unzipping JDG Modules ..."
       unzip -q ${JDG_MODULES_ZIP_PATH} -d setupOutput/jdg-modules
-      export JDG_MODULES=`cd setupOutput/jdg-modules/*;pwd`
+      export JDG_MODULES=`cd setupOutput/jdg-modules;pwd`
```

Comment:
are you sure this changes are necessary ? because this does not fail in testing environements

Reply:
I can only go from what occurred on my machine, but without them the JDG modules were not being copied to the server modules dir

Reply:
@karesti You were right these changes weren't necessary. The problem was that I used the modules zip built straight from source, which does not have the `modules` and `docs` subdirectories, hence why my change was needed on my machine. When using the released zip everything was fine, so I have removed the commit from this PR. Sorry for the confusion.